source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/workflow/signalr.jsx

```diff
@@ -1,10 +1,13 @@
 import * as signalR from '@aspnet/signalr';
```

Comment:
this is mostly copied from DWKit's source. modified a bit to be less verbose in the logging, and then also auth via jwt